src/Context/SoapContext.php

```diff
@@ -273,4 +273,15 @@ public function beforeStepCheckForException(BeforeStepScope $scope)
             throw $this->fault;
         }
     }
+
+    /**
+     * @Given I register the following headers:
```

Comment:
Would it be better if we would use "@Given I register the following SOAP headers:"? to avoid potential collision with other libraries and for consistency with other steps?